### ISSUE TITLE
Melon block still not fixed

### ISSUE THREAD

**Issue by Elric02** [Zrips/Jobs#562]
**Description of issue:**

melon_block isn't recognized when /jobs reloading

---

**ERROR:**
```
> [21:36:29 WARN]: [Jobs] Job Farmer has an invalid Break type property: melon_block!
```

**CONFIG SECTION:**
```
    Break:
      melon_block:
        income: 0.4
        experience: 1
```

---

**Server Type (Spigot/Paperspigot/etc):** 

Spigot

**Server Version (using `/ver`):** 

1.12.2

EDIT: I'm running the latest version of Jobs Reborn (4.13.1)

**Comment by montlikadani** [comment]
Replace `melon_block` to `melon`

**Comment by Elric02** [comment]
not working neither
`> [23:21:44 WARN]: [Jobs] Job Farmer has an invalid Break type property: MELON (MELON)! Material must be a block!`

**Comment by mrdollardx44** [comment]
still need a fix to this bug please !

**Comment by IAmSoccer** [comment]
> still need a fix to this bug please !


Hey, I'm not an actual support person, but here is a fix.

```
    legacy_melon_block:
      income: 1
      points: 1
      experience: 1
```
      
      
      So use "legacy_melon_block" in place of "melon" or "melon_block" for 1.12.2 should work.

**Comment by SrLeojaco** [comment]
> > still need a fix to this bug please !
> 
> Hey, I'm not an actual support person, but here is a fix.
> 
> ```
>     legacy_melon_block:
>       income: 1
>       points: 1
>       experience: 1
> ```
> 
> ```
>   So use "legacy_melon_block" in place of "melon" or "melon_block" for 1.12.2 should work.
> ```

T-H-A-N-K-S. I don't know if recent versions were fixed but i have been looking for this for 1.8 for a looooooong time.
